api/RestfulService.php

```diff
@@ -175,7 +191,7 @@ public function request($subURL = '', $method = "GET", $data = null, $headers =
 			$method,
 			$data,
 			array_merge((array)$this->customHeaders, (array)$headers),
-			$curlOptions + (array)$this->config()->default_curl_options,
+			$curlOptions + (array)$this->get_default_curl_options(),
```

Comment:
Nope, I didn't miss that per se. You cannot call an instance method statically but you can certainly call static methods from an instance context, which uses late static binding akin to `static` keyword, by default and it's easy :D It may be a code smell (I.e. Unexpected) but it's based on a solid principle at least. That's all from memory but I'm pretty certain it's fine, unless other parts of the file are calling static methodson itself differently (and I've frequently seen `self` in SS unfortunately).